.github/workflows/cygwin.yml

```diff
@@ -203,20 +203,14 @@ jobs:
           AUTOCONF: /usr/bin/autoconf-2.69
           MAKEFLAGS: dw
         run: |
+          export PATH="/usr/local/bin:$PATH"
           ccache -s
           git describe
-          cat <<EOT >> mplsetup.cfg
-          [rc_options]
-          backend=Agg
-
-          [libs]
-          system_freetype = False
-          system_qhull = True
-          EOT
-          cat mplsetup.cfg
           # All dependencies must have been pre-installed, so that the minver
           # constraints are held.
-          python -m pip install --no-deps -ve .
+          python -m pip install --no-deps --no-build-isolation --verbose \
+            --config-settings=setup-args="-DrcParams-backend=Agg" \
```

Comment:
Note that in the long commit message of e5e0e2fde61e62f56529d6d7d4ff1fa004343026, it was noted that system freetype/qhull were just for quicker testing and should probably have been reverted, so I have not carried that over to here.